docs/scos/user/intro-to-spryker/whats-new/php8-as-a-minimum-version-for-all-spryker-projects.md

```diff
@@ -0,0 +1,64 @@
+---
+title: PHP 8.0 as the minimum version for all Spryker projects
+description: In the coming weeks, Spryker will release a new version of its Demo Shops requiring PHP 8.0 as the minimum version.
+last_updated: November 14, 2022
+template: concept-topic-template
+---
+
+In the coming weeks, Spryker will release a new version of its Demo Shops requiring PHP 8.0 as the minimum version. PHP 7.4 will no longer be supported. Spryker's new module releases will only be compatible with PHP version 8.0 or later.
```

Comment:
```suggestion
In the coming weeks, Spryker will release a new version of its Demo Shops, which will require PHP 8.0 as their minimum PHP version. PHP 7.4 will no longer be supported. Spryker's new module releases will only be compatible with PHP version 8.0 or later.
```

Reply:
Thats old news, long long time like this